Makefile

```diff
@@ -519,9 +520,11 @@ docker-compose-sources: .git/hooks/pre-commit
 	    -e enable_ldap=$(LDAP) \
 	    -e enable_splunk=$(SPLUNK) \
 	    -e enable_prometheus=$(PROMETHEUS) \
-	    -e enable_grafana=$(GRAFANA) $(EXTRA_SOURCES_ANSIBLE_OPTS)
+	    -e enable_grafana=$(GRAFANA) $(EXTRA_SOURCES_ANSIBLE_OPTS) \
+	    -e dbname=$(DBNAME) \
 
 docker-compose: awx/projects docker-compose-sources
+	-docker rm tools_postgres_1
```

Comment:
this is a little weird. Can we do without it?

Reply:
I agree this is weird and tried to get it work without this. It seems to be a limitation of docker compose -- changing the volume information in the docker-compose.yml file doesn't seem to change the volume the container uses if that container already exists.

I may try to find another way to do this.